src/kepler_model/train/pipeline.py

```diff
@@ -241,6 +249,7 @@ def NewPipeline(
     isolator=MinIdleIsolator(),
     target_energy_sources=PowerSourceMap.keys(),
     valid_feature_groups=FeatureGroups.keys(),
+    use_vm_metrics=False,
```

Comment:
can you update the offline_trainer.py to pick up this new param from e.g. env var?